.github/workflows/codeql.yml

```diff
@@ -21,7 +21,7 @@ jobs:
 
          # Initializes the CodeQL tools for scanning.
          - name: Initialize CodeQL
-           uses: github/codeql-action/init@v1
+           uses: github/codeql-action/init@v2
```

Comment:
☕️💡 suggestion if you are updating this version then let’s make it compliant with all update: https://github.com/github/codeql-action

aka checkout to be at “v3”

![Uploading 38E2B938-527F-4EF9-90F5-E7E5A6FF0D44.jpeg…]()

Reply:
you're right, ive update it